warehouse/query-core/src/main/java/datawave/query/tables/ssdeep/SSDeepSimilarityQueryLogic.java

```diff
@@ -151,34 +142,17 @@ public void setupRanges(Query settings, SSDeepSimilarityQueryConfiguration confi
         }
 
         final Multimap<NGramTuple,SSDeepHash> queryMap = nGramEngine.preprocessQueries(queries);
-        final Set<Range> ranges = new TreeSet<>();
-
-        final IntegerEncoding bucketEncoder = new IntegerEncoding(config.getBucketEncodingBase(), config.getBucketEncodingLength());
-        final ChunkSizeEncoding chunkSizeEncoder = new ChunkSizeEncoding();
-
-        final int indexBuckets = config.getIndexBuckets();
 
         if (queryMap.isEmpty()) {
             String message = "Unable to generate SSDeepHash ngrams for query: " + settings.getQuery() + ", possibly due to invalid SSDeep hash(es)?";
             log.error(message);
             throw new SSDeepRuntimeQueryException(message);
         }
 
-        // TODO: stream?
-        for (NGramTuple ct : queryMap.keys()) {
-            final String sizeAndChunk = chunkSizeEncoder.encode(ct.getChunkSize()) + ct.getChunk();
-            for (int i = 0; i < indexBuckets; i++) {
-                final String bucketedSizeAndChunk = bucketEncoder.encode(i) + sizeAndChunk;
-                ranges.add(Range.exact(new Text(bucketedSizeAndChunk)));
-            }
-        }
-
-        log.info("Generated " + queryMap.size() + " SSDeepHash ngrams of size " + nGramEngine.getNgramSize() + " and " + ranges.size() + " ranges. ");
+        log.info("Generated " + queryMap.size() + " SSDeepHash ngrams of size " + nGramEngine.getNgramSize() + ".");
```

Comment:
Sorry I didn't catch this one in my initial review, but another candidate for slf4j / debug level?
```suggestion
        log.debug("Generated {} SSDeepHash ngrams of size {}.", queryMap.size(), nGramEngine.getNgramSize());
```
Happy to let this one go for later.

Reply:
I changed SimilarityQueryLogic logging to use sl4fj and parameterized logs in the most recent commit to revert the try block change while I figure out another approach to the try-with-resources warning.